messages.json

```diff
@@ -3,7 +3,7 @@
         "requestCode": "A0",
         "responseCode": "A1",
         "requestPattern": "mode:1/string, keyType:3/string, keySchemeLmk:1/string, deriveKeyMode:deriveKeyModeLen/string, dukptMasterKeyType:dukptMasterKeyTypeLen/string, dukptMasterKey:dukptMasterKeyLen/string, ksn:ksnLength/string, zkaMasterKeyType:zkaMasterKeyTypeLength/string, zkaMasterKey:zkaMasterKeyLength/string, zkaOption:zkaOptionLength/string, zkaRndi:zkaRndiLength/string, delimiter:delimiterLength/string, keyZmkTmkFlag:keyZmkTmkFlagLength/string, keyZmkTmk:keyZmkTmkLength/string, keySchemeZmkTmk:keySchemeZmkTmkLength/string, delimiterLmk:delimiterLmkLen/string, lmkIdentifier:lmkIdentifierLen/string, tr31BlockData:tr31BlockDataLen/string",
-        "responsePattern": "errorCode:2/string, key:33/string, rest/binary",
+        "responsePattern": "errorCode:2/string, rest/binary",
```

Comment:
Removing `key` from the response pattern will break the current and previous versions of the Payshield port. This change is only acceptable in a new major version of the codec. Same applies to the similar changes below.